charter.md

```diff
@@ -0,0 +1,34 @@
+#Charter for Working Group
+
+There are many situations in which it is desirable to share a digital credential with another person. For example, you may want to provide access to your vehicle to a Valet. You may also want to provide access to your home to your cat sitter. Or, you may want to share a hotel key with your spouse. Today, no such standardized method exists in a cross-platform, multidisciplinary capacity. 
+
+The WG charter includes the definition and standardization of a protocol that will facilitate such credential transfers from individual to individual. The protocol will leverage a “relay server” to transfer data from sender to recipient. The scope of the transfer is limited to a single origin device and a single destination device.
+
+Privacy goals include:
+
+- The relay server should not see sensitive details of the share
+- The relay server should not be able to provision the credential itself, acting as an intermediary for the recipient (MiTM)
+- The relay server should not know the identity of the sender nor receiver
```

Comment:
I don't think this rule always applies to the sender. Because the sender client and relay server are both managed by the same organization, where the identity is already known in the ecosystem in most cases. The spec should allow proprietary oauth built on top of this spec for sender and relay communication

Reply:
The spirit of this goal is to make sure that the relay server doesn't try to know identities of sender and receiver. 
In theory sender client and relay server can be managed by separate organizations. 
Even in case where they are managed by same org the spirit of the goal should apply.

Reply:
I believe Crystal's comment is valid - we may want to change wording from "should not know the identity" to something like "should not store/track the identity"

Reply:
changing the wording to "relay server should not persist the identity of the sender nor receiver"